### ISSUE TITLE
Getting oban_pro in github ci stopped working

### ISSUE THREAD

**Issue by barnabasJ** [sorentwo/oban#1096]
About 5 days ago our install dependency step in github ci stopped working with the following error

```
 Failed to fetch record for oban/oban_pro from registry (using cache instead)
:timeout
** (Mix) Unknown package oban_pro in lockfile
```

We didn't change the version. I tried deleting all the caches, updating the oban_pro version. But nothing worked.

Installing locally still works.

**Comment by sorentwo** [comment]
There is ongoing discussion about this in the #oban and #hex channels on Slack. Nothing changed on our side for hosting, and this is what we've deduced so far:

- It’s not a problem with recent hex versions (initially we wondered if it was from v2.1.1)
- It’s not an IPv4/IPv6 issue (it seemed like it was due to an IPv6 issue, that was red herring)
- It’s not a problem with the our server (requests are served normally, logs show only some of the package requests coming through)
- It's not an issue with Docker vs the runner (there are timeout failures in either environment)

The problem appears to be in the networking between GH and Fly. Only one of those is under our control, and we’re standing up a replica on another network to verify.
